server/views/serviceProviderReferrals/checkAssignment.njk

```diff
@@ -21,7 +21,13 @@
       <h2 class="govuk-heading-m">Caseworker</h2>
 
       {{ govukSummaryList(summaryListArgs) }}
-      {{ govukButton({ text: "Confirm assignment" }) }}
+      <form method="post" action={{ presenter.formAction }}>
+        {% for name, value in presenter.hiddenFields %}
+          <input type="hidden" name={{ name }} value={{ value }}>
```

Comment:
Would be nice to add a bit more detail to the commit description as to what this is for.

Reply:
have done this